decidim-admin/app/controllers/decidim/admin/participatory_process_step_activations_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+require_dependency "decidim/admin/application_controller"
+
+module Decidim
+  module Admin
+    # Controller that allows managing all the Admins.
+    #
+    class ParticipatoryProcessStepActivationsController < ApplicationController
+      def create
+        authorize ParticipatoryProcessStep
+
+        ActivateParticipatoryProcessStep.call(process_step) do
+          on(:ok) do
+            flash[:notice] = I18n.t("participatory_process_step_activations.create.success", scope: "decidim.admin")
```

Comment:
I'm starting to think we should create a generic `t` method on controllers and views that automatically prepends lowercased module names on the scope.